tests/api/test_conversation_export.py

```
@@ -8,10 +8,12 @@
 import urllib.request
 import urllib.error
 import json
-from typing import Any, Dict, Optional
+from typing import Any, Dict
```

Ruff (0.15.20)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)